packages/fxa-auth-server/fxa-oauth-server/lib/db/mysql/index.js

```diff
@@ -469,15 +472,19 @@ MysqlStore.prototype = {
     return this._write(QUERY_CODE_DELETE, [hash]);
   },
   generateAccessToken: function generateAccessToken(vals) {
+    // In certain scenarios to help avoid thundering herd, we add some padding to the expiration time of the token.
+    vals.ttl =
```

Comment:
Maybe not a real problem and just a nit, but modifying incoming params in a function always looks like future side-effect trouble to me. Might be better to assign to a new const var and use that below.